docs/api/v1/api.md

```diff
@@ -0,0 +1,833 @@
+                ]
+            }, 
+            "codeScan": {
+                "command": [
+                    "string"
+                ], 
+                "outputs": [
+                    "string"
+                ]
+            }, 
+            "imageBuild": {
+                "buildInfos": [
+                    {
+                        "contextDir": "string", 
+                        "dockerfile": "string", 
+                        "dockerfilePath": "string", 
+                        "imageName": "string"
+                    }
+                ]
+            }, 
+            "imageRelease": {
+                "releasePolicy": [
+                    {
+                        "image": "string", 
+                        "type": "string"                // type of image release policy, it must be Always or Never or IntegrationTestSuccess.
+                    }
+                ]
+            }, 
+            "integrationTest": {
+                "integrationTestSet": {
+                    "command": [
+                        "string"
+                    ], 
+                    "image": "string"
+                }, 
+                "services": [
+                    {
+                        "command": [
+                            "string"
+                        ], 
+                        "envVars": [
+                            {
+                                "name": "string", 
+                                "value": "string"
+                            }
+                        ], 
+                        "image": "string", 
+                        "name": "string"
+                    }
+                ]
+            }, 
+            "package": {
+                "command": [
+                    "string"
+                ], 
+                "outputs": [
+                    "string"
+                ]
+            }, 
+            "unitTest": {
+                "command": [
+                    "string"
+                ], 
+                "outputs": [
+                    "string"
+                ]
+            }
+        }
+    }, 
+    "createdTime": "2017-08-23T08:40:33.764Z", 
+    "description": "string", 
+    "id": "string", 
+    "name": "string", 
+    "notification": {
+        "emailNotification": {
+            "emails": [
+                "string"
+            ]
+        }, 
+        "notificationPolicy": "string"
+    }, 
+    "owner": "string", 
+    "projectID": "string", 
+    "serviceID": "string", 
+    "updatedTime": "2017-08-23T08:40:33.764Z"
+}
+```
+
+Pipeline is responsible for automating the lifecycle management of an application, and can safely and reliably deploy the application from the source code to the production environment in strict accordance with a set of scientific and rational software management processes.
+
+### Add a pipeline
+
+Create a new pipeline.
+
+#### Request
+
+Http method: `POST`
+
+URL: `/api/v1/projects/{project}/pipelines`
+
+Body:
+
+```json
+{
```

Comment:
Just put `PipelineDataStructure` here like the update API, instead of example data.

Reply:
done